app/models/manageiq/providers/container_manager/container_template.rb

```diff
@@ -0,0 +1,67 @@
+class ManageIQ::Providers::ContainerManager::ContainerTemplate < ContainerTemplate
```

Comment:
Shared code for kubernetes and openshift should go into manageiq-providers-kubernetes.